src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/operations/acl.py

```diff
@@ -0,0 +1,98 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.util import CLIError
+from azure.cli.core.profiles import get_sdk, ResourceType
+
+
+def create_acl_policy(cmd, client, container_name, policy_name, start=None, expiry=None, permission=None, **kwargs):
+    """Create a stored access policy on the containing object"""
+    t_access_policy = get_sdk(cmd.cli_ctx, ResourceType.DATA_STORAGE, 'AccessPolicy', mod='common.models') or get_sdk(
+        cmd.cli_ctx, ResourceType.DATA_STORAGE, 'AccessPolicy', mod='models')
+
+    acl = _get_acl(cmd, client, container_name, **kwargs)
+    acl[policy_name] = t_access_policy(permission, expiry, start)
+    if hasattr(acl, 'public_access'):
+        kwargs['public_access'] = getattr(acl, 'public_access')
+
+    return _set_acl(cmd, client, container_name, acl, **kwargs)
+
+
+def get_acl_policy(cmd, client, container_name, policy_name, **kwargs):
+    """Show a stored access policy on a containing object"""
+    acl = _get_acl(cmd, client, container_name, **kwargs)
+    return acl.get(policy_name)
+
+
+def list_acl_policies(cmd, client, container_name, **kwargs):
+    """List stored access policies on a containing object"""
+    return _get_acl(cmd, client, container_name, **kwargs)
+
+
+def set_acl_policy(cmd, client, container_name, policy_name, start=None, expiry=None, permission=None, **kwargs):
+    """Set a stored access policy on a containing object"""
+    if not (start or expiry or permission):
+        raise CLIError('Must specify at least one property when updating an access policy.')
+
+    acl = _get_acl(cmd, client, container_name, **kwargs)
+    try:
+        policy = acl[policy_name]
+        policy.start = start or policy.start
+        policy.expiry = expiry or policy.expiry
+        policy.permission = permission or policy.permission
+        if hasattr(acl, 'public_access'):
+            kwargs['public_access'] = getattr(acl, 'public_access')
+
+    except KeyError:
+        raise CLIError('ACL does not contain {}'.format(policy_name))
+    return _set_acl(cmd, client, container_name, acl, **kwargs)
+
+
+def delete_acl_policy(cmd, client, container_name, policy_name, **kwargs):
+    """ Delete a stored access policy on a containing object """
+    acl = _get_acl(cmd, client, container_name, **kwargs)
+    del acl[policy_name]
+    if hasattr(acl, 'public_access'):
+        kwargs['public_access'] = getattr(acl, 'public_access')
+
+    return _set_acl(cmd, client, container_name, acl, **kwargs)
+
+
+def _get_service_container_type(cmd, client):
+    from ..sdkutil import get_table_data_type
+    t_block_blob_svc, t_file_svc, t_queue_svc = get_sdk(
```

Comment:
Why does `cmd.get_models` not work here?

Reply:
Because for storage multiapi package the models are in different namespaces. For example, `blob.modes` and `file.models`.